.github/CONTRIBUTING.md

```diff
@@ -8,6 +8,13 @@ request, follow the [steps](#pull-request-workflow) below. We follow a
 [code of conduct](CODE_OF_CONDUCT.md) as our guide for community
 behavior.
 
+## Environment Variables
+
+We use the [Abstract SDK](https://sdk.goabstract.com/) to source versioned
```

Comment:
Avoid first person pronoun usage. Prefer "The website uses..." or "Garden uses..." vs "We use..."